setup.py

```diff
@@ -0,0 +1,27 @@
+from setuptools import setup
+
+with open('README.md') as fp:
+    readme = fp.read()
+
+with open('requirements.txt') as f:
+    install_requires = f.read().splitlines()
+
+setup(name='econtools',
```

Comment:
all these options:
<img src="http://s2.quickmeme.com/img/bc/bc75b955f736ac0cb29ee79aee4bf206103945487a1449dcecdc993d688f1cc1.jpg" width=200 />

Reply:
I would change the version value https://github.com/elifesciences/econ-tools/blob/d91a41a2054ce5408ea6d46b3027a8ad3453be24/setup.py#L10

to be an imported value from the econtools/__init__.py: `__version__ = '0.0.1'`. It is the standard way, allows the version to be checked by users and keeps the `setup.py` up to date when the version gets bumped.

Reply:
This email does not seem to work:
https://github.com/elifesciences/econ-tools/blob/d91a41a2054ce5408ea6d46b3027a8ad3453be24/setup.py#L18
I changed it to the tech team address in the last setup.py file I did:
`maintainer_email='tech-team@elifesciences.org',`

Reply:
Is this going on PyPi?